src/status_im/contexts/wallet/create_account/edit_derivation_path/view.cljs

```diff
@@ -38,83 +73,106 @@
 
-       (case @state
-         :default
-         [quo/bottom-actions
-          {:theme            theme
-           :actions          :one-action
-           :button-one-label (i18n/label :t/reveal-address)
-           :button-one-props {:type      :outline
-                              :icon-left :i/keycard-card
-                              :on-press  reveal-action}}]
+         (case @state
+           :default
+           [quo/bottom-actions
+            {:theme            theme
+             :actions          :one-action
+             :button-one-label (i18n/label :t/reveal-address)
+             :button-one-props {:type      :outline
+                                :icon-left :i/keycard-card
+                                :on-press  reveal-action}}]
 
-         :empty
-         [quo/bottom-actions
-          {:theme            theme
-           :actions          :one-action
-           :button-one-label (i18n/label :t/derive-addresses)
-           :button-one-props {:type      :outline
-                              :icon-left :i/keycard-card
-                              :on-press  derive-action}}]
+           :empty
+           [quo/bottom-actions
+            {:theme            theme
+             :actions          :one-action
+             :button-one-label (i18n/label :t/derive-addresses)
+             :button-one-props {:type      :outline
+                                :icon-left :i/keycard-card
+                                :on-press  derive-action}}]
 
-         :show
-         [rn/view {:style style/revealed-address-container}
-          [rn/view {:style (style/revealed-address theme)}
-           [quo/text
-            {:weight :monospace}
-            temp/address]]
-          [quo/info-message
-           {:type  :success
-            :icon  :i/done
-            :style style/info}
-           (i18n/label :t/address-activity)]]
+           :show
+           [rn/view {:style style/revealed-address-container}
+            [rn/view {:style (style/revealed-address theme)}
+             [quo/text
+              {:weight :monospace}
+              temp/address]]
+            [quo/info-message
+             {:type  :success
+              :icon  :i/done
+              :style style/info}
+             (i18n/label :t/address-activity)]]
 
-         :choose
-         [rn/view {:style style/temporal-placeholder}
-          [quo/text "Dropdown input will be here"]
-          [quo/button
-           {:on-press (fn [_]
-                        (reset! path-value (utils/get-formatted-derivation-path 1))
-                        (choose-action))} "Choose"]]
-         nil)
+           :choose
+           [rn/view {:style style/temporal-placeholder}
+            [quo/text "Dropdown input will be here"]
+            [quo/button
+             {:on-press (fn [_]
+                          (reset! path-value (utils/get-formatted-derivation-path 1))
+                          (choose-action))} "Choose"]]
```

Comment:
formatting is strange here, can you move "Choose" to a new line?
and probably we should i18n this string

Reply:
> probably we should i18n this string

@J-Son89 that's a temporary placeholder that was there